docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -86,8 +118,48 @@ \section{Team Meeting Plan}
 
 \section{Team Communication Plan}
 
+\section{Communication Plan}
+
+1.	Primary Liaison for External Stakeholders: Kartik will act as the main point of contact for external stakeholders, ensuring clear communication with groups like Air Rescue Services and other key parties. \\
+\\
+
+2.	Managing Cross-Team Communication: Responsible for ensuring seamless communication between team members and supervisor during any conflicts of interest, Kartik will coordinate updates, ensuring everyone is aligned with project objectives. \\
+\\
+
+3.	Issue Escalation and Resolution: Monitoring GitHub issues, any critical roadblocks will be escalated and resolved through team discussions during meetings or via Teams, ensuring the project stays on track.\\
+\\
+
+4.	Feedback Coordination: Feedback from external stakeholders will be compiled and tracked to ensure it is clearly communicated to the development team and integrated into project improvements.\\
+\\
+
+5.	Regular Stakeholder Updates: Team members will manage the creation and dissemination of concise updates to stakeholders, keeping them informed on project progress, timelines, and deliverables. \\
+\\
+
+6.	Documentation of Key Communications: All significant external communications and feedback will be documented for future reference, maintaining transparency and accountability across the project.\\
+\\
+
 \wss{Issues on GitHub should be part of your communication plan.}
 
+\section{Git Workflow}
```

Comment:
I think this section needs more detail:
![image](https://github.com/user-attachments/assets/a7aa6299-df91-45b6-8b31-41ade2e9369c)

Seems like we need to cover a more step by step guide, as well as specific descriptions for labels (eg. meeting tag for meetings, documentation tag for docs)

Reply:
I have updated the workflow plan to meet the rubric requirements.